docs/docs/50-configuration/99-encryption-providers/00-overview.md

```diff
@@ -0,0 +1,92 @@
+
+## Supported Encryption Providers
+
+Obot supports the following encryption providers:
+
+1. [AWS KMS](./01-aws-kms.md)
+2. [Azure Key Vault](./02-azure-key-vault.md)
+3. [Google Cloud KMS](./03-google-cloud-kms.md)
+4. [Custom](./04-custom-provider.md)
+
+## How Encryption Works
+
+Obot uses the Kubernetes EncryptionConfiguration format to encrypt data at rest. The encryption provider:
+
+1. Receives data to encrypt via a Unix socket connection
+2. Encrypts the data using the configured KMS provider
+3. Returns the encrypted data to be stored in the database
+4. On retrieval, decrypts the data before returning it to the application
+
+All encrypted string fields are base64-encoded after encryption for safe storage.
+
+## Encrypted Resources
+
+When you enable an encryption provider, the following resource types are automatically encrypted:
+
+| Resource Type | Description |
+|---------------|-------------|
+| `credentials` | Credential store data |
+| `runstates.obot.obot.ai` | Run state data for agent executions |
+| `users.obot.obot.ai` | User account information |
+| `identities.obot.obot.ai` | Identity provider data |
+| `mcpoauthtokens.obot.obot.ai` | MCP OAuth tokens |
+| `mcpauditlogs.obot.obot.ai` | MCP audit log data |
+| `sessioncookies.obot.obot.ai` | Session cookie data |
+
+## Complete List of Encrypted Fields
+
+### User Data (`users.obot.obot.ai`)
+All personal user information is encrypted:
+- `username` - User's username
+- `email` - User's email address
+- `displayName` - User's display name
+- `iconURL` - User's profile icon URL
+- `originalEmail` - Original email from identity provider
+- `originalUsername` - Original username from identity provider
+
+### Identity Data (`identities.obot.obot.ai`)
+Identity provider information is encrypted:
+- `providerUsername` - Username from identity provider
+- `email` - Email from identity provider
+- `providerUserID` - User ID from identity provider
+- `iconURL` - Icon URL from identity provider
+
+### MCP OAuth Tokens (`mcpoauthtokens.obot.obot.ai`)
+All OAuth-related secrets are encrypted:
+- `accessToken` - OAuth access token
+- `refreshToken` - OAuth refresh token
+- `clientID` - OAuth client ID
+- `clientSecret` - OAuth client secret
+- `state` - OAuth state parameter
+- `verifier` - PKCE verifier
+
+### Session Cookies (`sessioncookies.obot.obot.ai`)
+Session authentication data is encrypted:
+- `cookie` - Session cookie value
+
+### MCP Audit Logs (`mcpauditlogs.obot.obot.ai`)
+Complete request/response data in audit logs is encrypted:
+- `requestBody` - HTTP request body (JSON)
+- `responseBody` - HTTP response body (JSON)
+- `requestHeaders` - HTTP request headers (JSON)
+- `responseHeaders` - HTTP response headers (JSON)
+
+> **Note**: Logs can be retrieved without request/response data using the `WithRequestAndResponse` flag.
```

Comment:
This should be removed. It's an implementation detail that allows us to hide this information for non-auditor users.